src/es6Remx/connect.js

```diff
@@ -1,11 +1,11 @@
-import * as _ from 'lodash';
+import isFunction from 'lodash.isfunction';
 import React from 'react';
-import { observer } from '../mobxReactClone'; // should import from mobx-react/custom when they fix issue #319
+import { observer } from 'mobx-react/custom'; // should import from mobx-react/custom when they fix issue #319
```

Comment:
I am not sure we can remove the mobx-react clone. there was a problem with multiple mobx instances (see relevant issue), and I don't think they fixed it. maybe we can update the mobx-react version of the clone.

Reply:
I've added `isolateGlobalState: true` in this PR, isn't that what you were looking for?

Reply:
Also, mobx & mobx-react are now bundled into a single file by webpack, so there’s no way that mobx-react will get a different mobx instance